session-2.tex

```diff
@@ -46,30 +46,30 @@
   \section{Team Tactics}
   \begin{frame}{General Tactics}
     \begin{itemize}
-      \item Know each-others strength and weaknesses like:
+      \item Know each other's strengths and weaknesses like:
       \begin{itemize}
-        \item type of problems (math, geometry, search, strings, graphs etc.)
+        \item types of problems (math, geometry, search, strings, graphs, etc.)
         \item debugging skills
         \item coding speed and accuracy
       \end{itemize}
-      \item parallelize
-      \item work on paper (e.g. pseudocode of flow diagram)
-      \item debug on paper
-      \item use cardboard dog principle when stuck
+      \item Parallelize
+      \item Work on paper (e.g. pseudocode or flow diagrams)
+      \item Debug on paper
+      \item Use cardboard dog principle when stuck
```

Comment:
Is there a reason why you wrote "cardboard dog principle" instead of "rubber duck debugging"? :stuck_out_tongue: :duck:

```suggestion
      \item Use rubber duck debugging when stuck
```